docs/api/apiv3/components/schemas/oauth_client_credentials_read_model.yml

```diff
@@ -0,0 +1,28 @@
+# Schema: OAuthClientCredentialsReadModel
+---
+type: object
+required:
+  - id
+  - _type
+  - clientId
+  - confidential
+properties:
+  id:
+    type: integer
+    minimum: 1
+  _type:
+    type: string
+    enum:
+      - OauthClientCredentials
+  clientId:
+    type: string
+    description: OAuth 2 client id
+  confidential:
+    type: boolean
+    description: true, if OAuth 2 credentials are confidential, false, if no secret is stored
```

Comment:
At least not from an OAuth perspective. It might be a completely valid OAuth 2 connection, with only a client id transported. The client id usually is not considered confidential. The purpose here is merely to verify a specific scope, a client accesses a server.

Reply:
FYI: moved to a later point in time, until we actually need to create non-confidential client credentials